.gitignore

```diff
@@ -62,3 +62,6 @@ PMMain/Midas/MidasGUI/GeneratedFiles/
 
 
 
+PMMain/Midas/MidasGUI/MidasGUI.vcxproj.user
+PMMain/Midas/MidasGUI/MidasGUI.vcxproj.filters
+PMMain/Midas/MidasGUI/MidasGUI.vcxproj
```

Comment:
Please take these out. *.user is already in the file and the other 2 should not be added.